openwisp_notifications/handlers.py

```diff
@@ -164,6 +165,36 @@ def notify_handler(**kwargs):
     return notification_list
 
 
+def send_single_email(instance):
```

Comment:
Can we move this function to utils.py?  I would rename it to "send_notification_email" and add docstring to the function which explains that this function should be used when there's only one notification for the recipient.